openquake/qa_tests_data/scenario_risk/case_2/gmfs.csv

```diff
@@ -0,0 +1,1000 @@
+scenario-000,,0.17111 0.305128 0.604032
```

Comment:
That's a good idea but right now I do not want to do that, otherwise I would break the equivalent test on the engine.